packages/block-library/src/post-comments-form/edit.js

```diff
@@ -86,7 +83,7 @@ export default function PostCommentsFormEdit( {
 			warning = sprintf(
 				/* translators: 1: Post type (i.e. "post", "page") */
 				__(
-					'Post Comments Form block: Comments for this post type (%s) are not enabled.'
+					'Post Comments Form block: Comments are not enabled for this post type (%s).'
```

Comment:
I changed the word order in this message to be more consistent with the updated [per-item message](https://github.com/WordPress/gutenberg/pull/42131/files#diff-abf88ae09f89f6af7894b1f8480350eb820df74aff7e2b61490e93dd48c4722fR66).